public/index.html

```diff
@@ -2,9 +2,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="shortcut icon" href="/public/favicon.ico" />
```

Comment:
Just question. Why you removed this variable ?

Reply:
for urls testing purposes and forgot to put it back later - changed to previous already